homework3/hw3_variables_objects.py

```diff
@@ -0,0 +1,15 @@
+i="I love Python! "
```

Comment:
nit: add spaces before and after operator for more readability
```suggestion
i = "I love Python! "
```